$OpenBSD: patch-data_gnome-nettool_ui,v 1.1 2011/09/12 07:28:53 jasper Exp $

From cbd30cf35c3f5bf0ce2e211b92848de85acef11a Mon Sep 17 00:00:00 2001
From: Jean Schurger <jean@schurger.org>
Date: Wed, 13 Jul 2011 03:31:10 +0000
Subject: Fix ugliness when window is bigger than expected

GtkCombo in traceroute, port scan and whois looks bad if the
gnome-nettool window is bigger than expected.

--- data/gnome-nettool.ui.orig	Tue Jan 11 11:32:42 2011
+++ data/gnome-nettool.ui	Wed May 18 15:09:03 2011
@@ -1623,6 +1623,7 @@
                     <property name="visible">True</property>
                     <property name="border_width">6</property>
                     <property name="spacing">12</property>
+                    <property name="expand">False</property>
                     <child>
                       <object class="GtkHBox" id="hbox36">
                         <property name="visible">True</property>
@@ -1775,6 +1776,7 @@
                     <property name="visible">True</property>
                     <property name="border_width">6</property>
                     <property name="spacing">12</property>
+                    <property name="expand">False</property>
                     <child>
                       <object class="GtkHBox" id="hbox37">
                         <property name="visible">True</property>
@@ -2337,6 +2339,7 @@
                     <property name="visible">True</property>
                     <property name="border_width">6</property>
                     <property name="spacing">12</property>
+                    <property name="expand">False</property>
                     <child>
                       <object class="GtkHBox" id="hbox38">
                         <property name="visible">True</property>
